src/components/Accordion/Accordion.stories.tsx

```diff
@@ -31,9 +31,9 @@ const checkboxes = [
 export const accordionWithCheckboxes = () => (
     <Accordion
         id="accordion"
-        buttonOptions={{ id: "accordionBtn", content: <>Click to expand</> }}
+        accordionLabel="Click to expand"
```

Comment:
the checkboxes look broken in this story

Reply:
Checkboxes have been lookin' broke for a while, I'm not sure when that bug got introduced 😣 I'll fix it here though